### PR TITLE
Exchnage rate Conversion, Jumbotron update

### PR DESCRIPTION
**Pull request by khubaibshah786**
- Added Jumbotron image
- Added card for paired Conversion
- Added customer CSS to conversion card to overlap Jumbotron
- Added input fields for paired conversion
- Added new JavaScript file called CountryCodeMapping.js to store the currency code key and currency name value to show in selection fields
- Added Custom CSS to set the Jumbotron image
- Added Custom CSS to set jumbotron text colour
- Added convert image to know which is base currency and which is target currency
- Added Logic to allow user to convert Currency's using Exchange currency Api
- Added Exchange rate card underneath conversion card to show exchange rate
- Added Flag of target currency in conversion card to show what has been converted
- Added more images for jumbotron to choose from

### CODE REVIEW

File: assets/script.js [hunk 1]
```diff
@@ -37,13 +51,18 @@ fetch(pairedConversion)
...
     console.log('paird converstion',data)
     var baseCode = data.base_code
+    var currencyName = data.target_data.currency_name_short
+    var flagUrl = data.target_data.flag_url
+    var locale = data.target_data.locale
     var timeLastUpdateUnix = data.time_last_update_unix
     var timeLastUpdateUtc = data.time_last_update_utc
-    var conversionRates = data.conversion_rate
-
-    console.log(timeLastUpdateUnix)
```
Comment (dpaul93) [inline assets/script.js:44]: Remove console.log

File: assets/script.js [hunk 2]
```diff
@@ -37,13 +51,18 @@ fetch(pairedConversion)
   .then(function (data) {
     console.log('paird converstion',data)
```
Comment (dpaul93) [inline assets/script.js:52]: remove console.log

File: assets/script.js [hunk 3]
```diff
@@ -11,24 +11,38 @@ function standardConversion(apikey) {
       console.log('standard conversion', data);
```
Comment (dpaul93) [inline assets/script.js:11]: remove console.log

**Review by CNtervisi — APPROVED** [review]

File: assets/script.js [hunk 4]
```diff
@@ -37,13 +51,18 @@ fetch(pairedConversion)
...
+    var flagUrl = data.target_data.flag_url
+    var locale = data.target_data.locale
     var timeLastUpdateUnix = data.time_last_update_unix
     var timeLastUpdateUtc = data.time_last_update_utc
-    var conversionRates = data.conversion_rate
-
-    console.log(timeLastUpdateUnix)
+    var conversionRate = data.conversion_rate
+    var convertedRate = currencyAmount * conversionRate
+    console.log(flagUrl);
```
Comment (dpaul93) [inline assets/script.js:61]: remove console.log

**Review by dpaul93 — CHANGES_REQUESTED** [review]
please see review and re-push

**Comment by khubaibshah786** [comment]
> please see review and re-push

all done

**Review by dpaul93 — APPROVED** [review]
Thanks for changes. Accepted merge request.

**Pull request merged**